.editorconfig

```diff
@@ -140,6 +138,18 @@ csharp_style_pattern_matching_over_as_with_null_check = true:suggestion
 csharp_style_inlined_variable_declaration = true:suggestion
 csharp_style_throw_expression = true:suggestion
 csharp_style_conditional_delegate_call = true:suggestion
+csharp_style_deconstructed_variable_declaration = true:suggestion
+dotnet_style_object_initializer = true:suggestion
+dotnet_style_collection_initializer = true:suggestion
+dotnet_style_coalesce_expression = true:suggestion
+dotnet_style_null_propagation = true:suggestion
+dotnet_style_explicit_tuple_names = true:suggestion
+dotnet_style_prefer_inferred_tuple_names = true:suggestion
+dotnet_style_prefer_inferred_anonymous_type_member_names = true:suggestion
+dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggestion
+dotnet_style_prefer_conditional_expression_over_return = false
```

Comment:
This one doesn't work: https://github.com/dotnet/roslyn/issues/27197

Reply:
Try `false:none`?  Not sure how things work if you don't have a severity.

Reply:
Reading the code, `false` is allowed. I tried `false:none` it results in the same behavior. I spent ~30 minutes trying to figure this out and gave up. My small repro stopped occurring once I attached a debugger.